deroff/deroff_ucb.c

```diff
@@ -785,8 +785,7 @@ _C(void)
  *
  *	Macro table definitions
  */
-#define	reg	register
-static int	argconcat = 0;		/* concat arguments together (-me only) */
+#define	reg	static int	argconcat = 0;		/* concat arguments together (-me only) */
```

Comment:
We have a small problem here.
It should be this:

```suggestion
#define	reg	/* antiregister */
static int	argconcat = 0;		/* concat arguments together (-me only) */
```

Contrarily, it will not compile because of a syntax error.